features/363-glossary/glossary.md

```diff
@@ -0,0 +1,48 @@
+Glossary
+========
+
+In this document, we define a list of terms to be used in communication.
+
+Objective
+---------
+
+In various documents, we will use different terms to refer to concepts in Skygear:
+- Documentation
+- Discussions
+- Emails/Announcements
```

Comment:
I can forasee some conflict if the terms here are expected to be in Emails / Annoucements or other marketing communications.

e.g.:

* If Skygear Auth is wrong but Auth Gear is right -- marketing-wise we would want to make sure the term Skygear is on it...
* Skygear Platform are currently used in most marketing communications instead of Skygear Cluster

So do we want to align marketing terms or not?

Reply:
In my opinion, if the marketing materials targets at technical staff (e.g. developers), they'll most likely also read the documentation, and using multiple terms referring to the same thing would be confusing.

Your example (platform vs cluster) is exactly the motivation for this glossary when I was drafting the Getting Started guide. Is there any important difference between "Skygear Platform" & "Skygear Cluster"? When should we use one term over another? This document intends to clarify it.

This is just a very rough draft. I just opened this PR so we can start discussing about it, as discussed in product meeting, considering we'll need to get started on documentations very soon, and consistent writing style and terminology is very important.

Reply:
Noted, I will re-comment this spec assuming the terms will be used on marketing side too.